core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1364,6 +1368,7 @@ public class SqlStdOperatorTable extends ReflectiveSqlOperatorTable {
   /**
    * The <code>UNNEST WITH ORDINALITY</code> operator.
    */
+  @LibraryOperator(libraries = {}) // do not include in index
```

Comment:
Why do we need to remove it from the standard operators, is it a breaking change?

Reply:
The operators for `UNNEST` and `UNNEST WTH ORDINALITY` have the same name and syntax, and therefore clash when added to a map. The latter was never accessed by name anyway.